client/signup/validation-fieldset/test/index.jsx

```diff
@@ -22,7 +22,7 @@ describe( 'ValidationFieldset', () => {
 	test( 'should include a FormInputValidation only when errorMessages prop is set.', () => {
 		const wrapper = shallow( <ValidationFieldset /> );
 
-		expect( wrapper.find( 'FormInputValidation' ).isEmpty() ).to.be.true;
```

Comment:
How are these changes connected to the PR? They seem harmless, I'm just curious if necessary.

Reply:
These are fixes to errors being thrown. Made during my trial.

See https://github.com/Automattic/wp-calypso/pull/25762/commits/94ba9003421dee8db6001e3c418d6c199c2b4e7e